flink-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaConsumerTestBase.java

```diff
@@ -295,6 +295,7 @@ public void runAutoOffsetRetrievalAndCommitToKafka() throws Exception {
 		env.getConfig().setRestartStrategy(RestartStrategies.noRestart());
 		env.setParallelism(parallelism);
 		env.enableCheckpointing(200);
+		env.getCheckpointConfig().setTolerableCheckpointFailureNumber(Integer.MAX_VALUE);
```

Comment:
This one has been reported to Jira : https://issues.apache.org/jira/browse/FLINK-12542